exercises/concept/freelancer-rates/.docs/instructions.md

```diff
@@ -1,19 +1,13 @@
 # Instructions
 
-In this exercise you will be writing code to help a freelancer communicate with a project manager by providing a few utility functions to quickly calculate day- and month rates, optionally with a given discount.
-
-We first establish a few rules between the freelancer and the project manager:
-
-- The daily rate is 8 times the hourly rate;
-- A month has 22 billable days.
-
-If the freelancer bills the project manager per month or more, there is a discount applied. This can be handy if the project manager has a fixed budget.
-
-Discounts are modeled as fractional numbers (percentage) between `0.0` (`0%`, no discount) and `0.90` (`90%`, maximum discount).
+In this exercise you will be writing code to help a freelancer communicate with his clients by providing a few utility functions to quickly calculate his hiring costs for projects.
```

Comment:
"hiring costs" sounds like the freelancer hires someone else which is not meant here I think, maybe something like this could be used instead

```suggestion
In this exercise you will be writing code to help a freelancer communicate with his clients about the prices of certain projects. You will write a few utility functions to quickly calculate the costs for the clients.
```